envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/infrastructure/consul/ConsulWatcherConfig.kt

```diff
@@ -108,5 +108,5 @@ open class ConsulWatcherConfig {
     }
 
     @Bean
-    fun kotlinModule() = KotlinModule()
+    fun kotlinModule() = KotlinModule.Builder().build()
```

Comment:
why this change?

Reply:
after upgrading Andamio default constructor KotrlinModule() stopped working (and it was also deprecated before) so i changed it to Builder. all defaults are the same so the behaviour is the same